### DISCUSSION TITLE
WinMerge compare Folder: should change to next compare when one folder is missing from Another

### DISCUSSION THREAD

**Discussion by congtuannct** [WinMerge/winmerge#2067] [General]
WinMerge compare Folder: should change to next compare (folder or file) when folder is missing from another. When compare 2 folder contains so many files and sub folders, one folder is missing (not exists) on another folder but WinMerge still continue woking on that folder and sub folders. I think WinMerge should stop compare this folder and it's sub folders and move next to compare another folder/files.

**Comment by congtuannct** [comment]
![image](https://github.com/WinMerge/winmerge/assets/46589838/679c111f-eb7b-4b1a-841b-7f3be26deeef)

**Comment by sdottaka** [comment]
The Full Contents compare method and Quick Contents compare method have the ability to display information to be displayed in the Left/Right EOL column and Left/Right Encoding column. Therefore, even if the file is only on one side, it is read.
(Though these are not default display columns...)

The Binary Conetns compare method should be faster because it doesn't retrieve that information.
Also, if you uncheck the [Include unique subfolder contents] checkbox in the Compare/Folder category of the Options window as shown below, files in subfolders that only exist in one will not be enumerated, which should speed up the process.

![image](https://github.com/WinMerge/winmerge/assets/98126/d5448b1b-4fe1-442d-bfb3-8ab6ce2559c1)
